sw/host/opentitanlib/src/transport/chip_whisperer/mod.rs

```diff
@@ -166,6 +166,12 @@ impl<B: Board + 'static> Transport for ChipWhisperer<B> {
             usb.spi1_enable(false)?;
             usb.clear_bitstream()?;
             Ok(None)
+        } else if action.downcast_ref::<GetSam3xFwVersion>().is_some() {
+            let usb = self.device.borrow();
+            //let (x, y, z) = usb.get_firmware_version()?;
+            //println!("Version {:?}.{:?}.{:?}", x, y, z);
```

Comment:
Leftover comments, perhaps?

Reply:
Oops, meant to clean that up. Thanks. fixed.